src/frontend/src/pages/FinancePage/FinanceComponents/ReportRefundModal.tsx

```diff
@@ -17,7 +18,8 @@ const schema = yup.object().shape({
       if (!value) return false;
       return Math.floor(value * 100) === value * 100;
```

Comment:
can you update this to check the the difference between these two values is below a threshold. Right now a floating point error can cause this to return an error when it shouldn't. (unrelated to your ticket, just found it while I was testing)